src/Microsoft.Azure.WebJobs/QueueAttribute.cs

```diff
@@ -26,7 +26,7 @@ namespace Microsoft.Azure.WebJobs
     [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1813:AvoidUnsealedAttributes")]
     [AttributeUsage(AttributeTargets.Parameter)]
     [DebuggerDisplay("{QueueName,nq}")]
-    public class QueueAttribute : Attribute
+    public class QueueAttribute : StorageAccountAttribute
```

Comment:
Ideally, we want (and I think these are in priority order):
1. No breaking changes
2. no redundant attributes (having StorageAccount and Storage would be confusing and is unnecessary from a binding perspective)
3. good names 

I don't think we can get all 3.  The current PR does 1&2.